config.dist.php

```diff
@@ -2,11 +2,11 @@
 
 /* database connection details
  ******************************/
-$db['host'] = "localhost";
-$db['user'] = "phpipam";
-$db['pass'] = "phpipamadmin";
-$db['name'] = "phpipam";
-$db['port'] = 3306;
+$db['host'] = getenv('IPAM_HOST') ? getenv('IPAM_HOST') : "localhost";
```

Comment:
I used this instead of the shorthand one since it looks like you still are trying to support php 5.1.